src/Pos/Lrc/Consumers.hs

```diff
@@ -14,6 +18,8 @@ import           Pos.Ssc.Class.Workers (SscWorkersClass (sscLrcConsumers))
 import           Pos.Update.Lrc        (usLrcConsumer)
 
 allLrcConsumers
-    :: (LrcMode ssc m, SscWorkersClass ssc)
+    :: forall ssc m.
+       (LrcMode ssc m, SscWorkersClass ssc)
     => [LrcConsumer m]
-allLrcConsumers = [delegationLrcConsumer, usLrcConsumer] ++ untag sscLrcConsumers
+allLrcConsumers = [delegationLrcConsumer, usLrcConsumer] ++
+                  untag (sscLrcConsumers @ssc)
```

Comment:
`Tagged` doesn't solve the problem here because `ssc` isn't mentioned in `=> [LrcConsumer m]`. So, I'm going to remove `Tagged` here completely because it's useless.

(Though previously it *did* solve the problem... no idea why.)